packages/table/src/table.tsx

```diff
@@ -870,19 +870,18 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     private selectAll = () => {
+        const { enableFocus } = this.props;
+
         const selectionHandler = this.getEnabledSelectionHandler(RegionCardinality.FULL_TABLE);
         // clicking on upper left hand corner sets selection to "all"
         // regardless of current selection state (clicking twice does not deselect table)
         selectionHandler([Regions.table()]);
 
         // move the focus cell to the top left
-        const newFocusedCellCoordinates = Regions.getFocusCellCoordinatesFromRegion(Regions.table());
-        const fullFocusCellCoordinates: IFocusedCellCoordinates = {
-            col: newFocusedCellCoordinates.col,
-            focusSelectionIndex: 0,
-            row: newFocusedCellCoordinates.row,
-        };
-        this.handleFocus(fullFocusCellCoordinates);
+        if (enableFocus) {
```

Comment:
whoops, don't need this check, because it's handled in `handleFocus`. will undo.